src/poetry/console/commands/install.py

```diff
@@ -67,6 +67,13 @@ class InstallCommand(InstallerCommand):
             flag=False,
             multiple=True,
         ),
+        option(
```

Comment:
This option is dead and should be removed (given the move to using the pyproject.toml).